From 899a265d8c47e7b1c684cd4907dc24acb7456720 Mon Sep 17 00:00:00 2001
From: "Li, Yifan2" <yifan2.li@intel.com>
Date: Fri, 13 Apr 2018 16:24:18 +0800
Subject: [PATCH 1349/2302] Fix compiler warnings of SOF.

This commit fix all 3 compiler warnings in SOF code.
sound/soc/soc-core.c:2999:3: misleading-indentation
sound/soc/soc-core.c:3002:4: misleading-indentation
sound/soc/intel/boards/bxt_tdf8532.c:80:12: unused-function

Signed-off-by: Li, Yifan2 <yifan2.li@intel.com>
---
 sound/soc/intel/boards/bxt_tdf8532.c | 2 ++
 sound/soc/soc-core.c                 | 6 ++++--
 2 files changed, 6 insertions(+), 2 deletions(-)

diff --git a/sound/soc/intel/boards/bxt_tdf8532.c b/sound/soc/intel/boards/bxt_tdf8532.c
index c437dca..275ecd6 100644
--- a/sound/soc/intel/boards/bxt_tdf8532.c
+++ b/sound/soc/intel/boards/bxt_tdf8532.c
@@ -77,6 +77,7 @@ static const struct snd_soc_dapm_route broxton_tdf8532_map[] = {
 #endif
 };
 
+#if !IS_ENABLED(CONFIG_SND_SOC_SOF_INTEL)
 static int bxt_tdf8532_ssp2_fixup(struct snd_soc_pcm_runtime *rtd,
 				struct snd_pcm_hw_params *params)
 {
@@ -88,6 +89,7 @@ static int bxt_tdf8532_ssp2_fixup(struct snd_soc_pcm_runtime *rtd,
 
 	return 0;
 }
+#endif
 
 /* broxton digital audio interface glue - connects codec <--> CPU */
 static struct snd_soc_dai_link broxton_tdf8532_dais[] = {
diff --git a/sound/soc/soc-core.c b/sound/soc/soc-core.c
index d69c76a..e70fc37 100644
--- a/sound/soc/soc-core.c
+++ b/sound/soc/soc-core.c
@@ -2997,11 +2997,13 @@ int snd_soc_dai_program_stream_tag(struct snd_pcm_substream *substream,
 	for (i = 0; i < rtd->num_codecs; i++) {
 		codec_dai = rtd->codec_dais[i];
 		codec_dai_ops = codec_dai->driver->ops;
-		if (codec_dai_ops->program_stream_tag)
+		if (codec_dai_ops->program_stream_tag) {
 			ret = codec_dai_ops->program_stream_tag(substream,
 				codec_dai, stream_tag);
-			if (ret)
+		}
+			if (ret) {
 				return ret;
+			}
 	}
 	return ret;
 
-- 
2.7.4

